src/components/Tabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import Tabs from "./Tabs";
+
+export default {
+    title: "Tabs",
+    component: Tabs,
+};
+
+const tabItems = {
+    "1": <p>Some content for the first tab item...</p>,
```

Comment:
Instead of assigning these to props, it would be nice if there was a way to pass them as children.  It makes for a more easy to read, and more flexible component.  There's a way this is being done in List ( https://github.com/NYPL/nypl-design-system/blob/development/src/components/List/List.tsx ) Or, creating a subcomponent called tabItem so that you have a structure that looks like 

```
<Tabs props={}>
<TabItem> {Content 1} </TabItem> 
<TabItem> { Content 2} </TabItem>
(etc) 
</Tabs>
```

Reply:
Good idea; I'll take care of it in the next round :)